### ISSUE TITLE
Is there an apk package for mruby?

### ISSUE THREAD

**Issue by johnpittman** [mruby/mruby#2939]
Migrating to alpine linux for docker containers and my software isn't seeing mruby after doing a build install. I there an apk package to do apk add <mruby_package_name>?

**Comment by cremno** [comment]
No, I couldn't find one. For mruby *vendoring* is usually the way to go anyway.

**Comment by matz** [comment]
Closing — Alpine apk packaging question, not in mruby's tracker scope. Check the alpine packaging tooling or build from source via the standard rake flow.
